docs/config/README.md

```diff
@@ -163,7 +163,7 @@ Function for transforming header texts into slugs. This affects the ids/links ge
 - Type: `Object`
 - Default: `{ target: '_blank', rel: 'noopener noreferrer' }`
 
-The key and value pair will be added to `<a>` tags that points to an external link. The default option will open external links in a new window.
+A key and value pair that will be added to `<a>` tags that point to an external link. The default option will open external links in a new window.
```

Comment:
2 `that` here.

Reply:
Not sure what you mean...

Reply:
You added a that, causing this sentence to appear two that, it is not fluent...